### ISSUE TITLE
onboarding/tip/ to introduce new feature to users

### ISSUE THREAD

**Issue by thianhlu** [wordpress-mobile/WordPress-iOS-Editor#466]

**Comment by meremagee** [comment]
- tooltip outside of editor context to notify users that may not have gone to editor that there is a more user-friendly experience
- reusable for onboarding of other, future features
- associated with user so it’s only shown once

**Comment by thianhlu** [comment]
We are trying to achieve two things with this feature. We want to introduce active/existing users of the app to the new & improved editor by showing them some feature onboarding so this comes as no surprise. The other main goal is showing users who do not post but did upgrade the app an encouraging onboarding to see if we can get them posting again.

WIP - We can do this many different ways but here is an example of a really slim onboarding. We would show a different message based on if they are active or not. Ignore the placeholder content.

![](https://cloudup.com/cwp7xOyyhxv+)
* Note we would prob get rid of "Edit" tip we have implemented in this case.

Or we can do something larger with more information like 
![](https://cloudup.com/cpruH1zoLKB+)

Or we can do a full screen one but rather not since we prob want to reserve that for full new user onboarding.

cc @meremagee @bummytime

**Comment by thianhlu** [comment]
Ignore color of the tip as well, we can change that

**Comment by meremagee** [comment]
Nice!

My preference would be to show both to all users.
- tool tip on reader upon upgrade (dismissed on tap/nav)
- "new editor" + details overlay when you hit the new editor after upgrade (dismissed with alert conf)

I think the message is universal and helpful whether you use the editor regularly (gives the heads up you need not to feel lost) or if you haven't used the editor in a while (seeing something about a new experience might be the incentive you need to check it out). 

This would also eliminate some complexity around determining active/non-active users.

**Comment by thianhlu** [comment]
K chatted with @bummytime 

We are gonna ditch the tooltip/call to action version and just do the modal. Also, we are keeping the tip. updated mock to come

**Comment by thianhlu** [comment]
![](https://cloudup.com/clFfdNURoQs+)

@meremagee We will need better copy for this modal. Possibly new image depending on copy.

@bummytime We will be displaying this upon open of the app after upgrade. Do not show to new users who have visual editor already enabled. Also, keep edit tip in place.

**Comment by bummytime** [comment]
@thianhlu roger that!

**Comment by meremagee** [comment]
After playing with a few [drafts](https://cloudup.com/cC227Q01HTk) and getting feedback, let's go with something like this: 

**Title**: Brand new editor
**Text**: This version of the WordPress app includes a beautiful new visual editor. Try it out by creating a new post.
**Confirm/dismiss**: Great, thanks!

why?
- since a user may skim and exit before reading entirely, want title to be descriptive enough to capture meaning at a glance
- conveys visual editor updates without using jargon (media, html, rich text, etc)
- with translation/no translation in mind, keeping idioms to a minimum
- clear yet descriptive
- not mentioning option to disable in prompt bc html view of editor should be robust & stable enough for users partial to the legacy editor

**Comment by diegoreymendez** [comment]
@meremagee - Is there a download link for the pen icon?

**Comment by meremagee** [comment]
@diegoreymendez - waiting on image asset from @thianhlu. just put a placeholder for now :+1:

**Comment by thianhlu** [comment]
WIP let me know if you guys like it and I'll clean it up and make svg

![](https://cloudup.com/cYmkGZZF8UT)

**Comment by thianhlu** [comment]
![](https://cloudup.com/cYmkGZZF8UT+)

**Comment by meremagee** [comment]
works for me. 

since the navigation is changing as well, I actually like how @diegoreymendez used the "new post" button to help folks make that mental leap. thoughts @thianhlu?

![](https://cloudup.com/cycaSAgtrsZ+)

**Comment by thianhlu** [comment]
- The button is the same right? I'm ok with either. I would like to go the new asset cause it looks visually more appealing and we can use the same asset if we do an email.
- Is the overlay lighter that usual or is that the default one? If so, make the separator line a bit lighter. Is it possible to do 0.5px line?
- @meremagee For the copy...Instead of "This version of the WordPress app" can we try "This update of the Wordpress iOS app"

**Comment by meremagee** [comment]
_This WordPress iOS app update includes a beautiful new visual editor. Try it out by creating a new post._

thoughts?

**Comment by diegoreymendez** [comment]
Closed by mistake.  Reopening.

**Comment by diegoreymendez** [comment]
@thianhlu - Replies to your questions:

- The button icon is the same.  It was chosen as the temporary artwork, but it turned out to be nice that it refers to the button you should be pressing to follow the popup's instructions.
- The overlay should be the default but I'll double check it.  We should be able to make a 0.5px line.
- Since there's a question by @meremagee about this, please let me know guys if a change is needed.

Points 1 and 2 are ready to be worked on... for point 3 I'd need the final text.  Let me know @meremagee and @thianhlu  !

**Comment by thianhlu** [comment]
Awesome

1) Lets change popup image. Here is are the images up to 3x: https://cloudup.com/cUj6itrkwMx
2) Make the 0.5px line
3) Use the latest text by mere "The WordPress app for iOS now includes a beautiful new visual editor. Try it out by creating a new post."

Question: If the user updates WP iOS then opens the app, sees onboarding but immediately closes the app. Let's make sure they see the onboarding again until they take "Great, thanks"

**Comment by meremagee** [comment]
> Question: If the user updates WP iOS then opens the app, sees onboarding but immediately closes the app. Let's make sure they see the onboarding again until they take "Great, thanks"

new issue for this specifically https://github.com/wordpress-mobile/WordPress-iOS-Editor/issues/510

**Comment by diegoreymendez** [comment]
Fixed.  Will continue with the remaining requests from [this issue](https://github.com/wordpress-mobile/WordPress-iOS-Editor/issues/510) separately.
